src/docs/Components/TextInput/index.mdx

```diff
@@ -13,6 +13,12 @@ status:
 
 <Preview nude path="input-state" />
 
+## Variations
+
+We have two different sizes of text inputs to answer differents needs. You can use both cases to suit your interface the best. Just keep in mind touch devices when placing smaller elements near each others.
+
+<Preview path="input-sizes" />
```

Comment:
@tiloyi The small input has a height of 2mu (32px), on the preview it make 34px.
The baseline of the label is centered with the height of the input :
<img width="911" alt="Capture d’écran 2019-12-04 à 11 52 58" src="https://user-images.githubusercontent.com/43212270/70148531-a6f9ea80-16a6-11ea-8077-5ae11f7fd5ba.png">

Reply:
Fixed.